### PR TITLE
2 4 user creation

### PR DESCRIPTION
**Pull request by naphets123**

### CODE REVIEW

File: cypress/integration/2_4_CreateUser.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+context("User creation text", () => {
+    let Testname = "paul";
+    let Testgroup = "TestUserGroup_User";
+    let Testaddress = "paul@paul.co";
+    let Alt_org_name = "Volunteer Hans";
```
Comment (DurkoMatko) [inline cypress/integration/2_4_CreateUser.js]: Nothing serious but the variable naming conventions don't match..as mentioned earlier, I wouldn't bother, but all the senior devs at work always tell me unify names under same convention :D

File: cypress/integration/2_4_CreateUser.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    beforeEach(function() {
+        cy.loginAsAdmin();
+        cy.visit('/?action=cms_users_edit&origin=cms_users');
+        
+    })
+
+    it("2_4_0_A Check empty form + empty submit",() => {
+        cy.get("input[data-testid='name_id']").should("be.visible")
+        cy.get("input[data-testid='email_id']").should("be.visible")
+        cy.get("span[id='select2-chosen-1']").should("be.visible")
```
Comment (DurkoMatko) [inline cypress/integration/2_4_CreateUser.js]: As mentioned in previous PRs, having some helper methods would increase readability of the code. Especially if the selector isn't clear. when using data-testid, it's quite readable, but e.g. `select2-chosen-1` is hard to understand.

File: cypress/integration/2_4_CreateUser.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    })
+
+    it("2_4_0_B Create New user",() => {
+        FillForm(Testname,Testaddress,Testgroup)
+        cy.get("button").contains("Save and close").click()
+        cy.get("tr").contains(Testaddress).should("be.visible")
+        DeleteTestUser(Testaddress)
+    })
+
+    it("2_4_1 Create New User without axpiry date", () => {
```
Comment (DurkoMatko) [inline cypress/integration/2_4_CreateUser.js]: "expiry date"

**Comment by HaGuesto** [comment]
close since PR is moved to a new branch

**Pull request closed without merging**